config/settings.yml

```diff
@@ -3,9 +3,43 @@
   :ems_hawkular:
     :event_handling:
       :event_groups:
+:ems_refresh:
+  :hawkular:
+    :refresh_interval: 15.minutes
+:event_handling:
```

Comment:
this has to be under `:ems -> :ems_hawkular`
check the settings.yml in the [amazon provider](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/config/settings.yml#L4) for reference